lib/erb_lint/rake/task.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "rake"
+require "rake/tasklib"
+require "erb_lint/cli"
+
+module ERBLint
+  module Rake
+    # Defines Rake tasks for use.
+    class Task < ::Rake::TaskLib
+      LOGGER = Logger.new($stdout, formatter: ->(_severity, _at, _name, message) { "#{message}\n" })
```

Comment:
I don't think this is necessary. Let's just use `puts` instead, or maybe [`Rake.rake_output_message`](https://ruby.github.io/rake/Rake/FileUtilsExt.html#method-i-rake_output_message) ?